pkg/konfluxgen/konfluxgen.go

```diff
@@ -324,6 +326,12 @@ func Generate(cfg Config) error {
 				r.Hermetic = "false"
 			}
 
+			if cfg.ECPolicyConfigName == "" {
+				r.ECPolicyConfiguration = "rhtap-releng-tenant/tmp-onboard-policy"
```

Comment:
I've created https://issues.redhat.com/browse/SRVCOM-3391 as a follow up on this policy